aiven/user_config.go

```diff
@@ -302,7 +302,10 @@ func convertAPIUserConfigToTerraformCompatibleFormat(
 			case string:
 				terraformConfig[key] = apiValue
 			case bool:
-				terraformConfig[key] = strconv.FormatBool(apiValue.(bool))
+				_, ok := schemaDefinition["api_type"]
+				if ok {
```

Comment:
I'm not sure I follow this logic. If api_type value is not present then we just silently don't set any value here?

Reply:
sry, I did partial push, added WIP to this PR